README.md

```diff
@@ -77,17 +77,6 @@ spec:
 EOF
 ```
 
-## Deploy authorino operator using operator-sdk
-1. Install operator-sdk bin
-   ```sh
-   make operator-sdk
-   ```
-2. Run operator-sdk bundle command 
-   ```
-   ./bin/operator-sdk run bundle quay.io/kuadrant/authorino-operator-bundle:latest
-   ```
```

Comment:
Why removing these instructions?

Reply:
Initially catalog image support was not there for s390x and ppc64le, so this instruction was added for user to install operator using operator sdk. Now since we have added catalog support, so user can follow the generic approach.